modules/ROOT/pages/whats-new-in-dw.adoc

```diff
@@ -0,0 +1,64 @@
+= What's New in DataWeave
+
+The 2.4 version of DataWeave introduces the following features:
+
+* Ability to read larger-than-memory strings automatically. When you are using the indexed reader strategy and processing a String with a size larger than 1.5 MB, DataWeave automatically splits the value in chunks to avoid out-of-memory issues. Related reader properties `com.mulesoft.dw.max_memory_allocation` and `com.mulesoft.dw.buffered_char_sequence.enabled`. See xref:dataweave-formats.adoc[Supported Data Formats].
+* New reader and writer properties for working with data formats. For example, the XML writer now enables you to define namespaces at the root level. 
+* Many new modules, functions, types, annotations, and variables. Some are experimental and subject to change in future releases.
+* A number of functions are overloaded to introduce new helper functions for
+handling `null` values.
```

Comment:
```suggestion
managing `null` values.
```

Reply:
Per Kim suggestion

Reply:
I think in this specific context, it's more common to say "handling."